lib/money-rails/configuration.rb

```diff
@@ -37,7 +37,7 @@ def register_currency=(currency_options)
     end
 
     def set_amount_column_for_default_currency!
-      amount_column.merge! postfix: "_#{default_currency.subunit.downcase.pluralize}" if default_currency.subunit
+      amount_column.merge! postfix: "_cents" if default_currency.subunit
```

Comment:
I guess this method isn't needed anymore since it's redefining an already present value